app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -1003,6 +1003,16 @@ def build_toolbar_disable_button(id)
       when "container_group_timeline"
         return "No Timeline data has been collected for this Pod" unless @record.has_events? || @record.has_events?(:policy_events)
       end
+    when "ManageIQ::Providers::Kubernetes::ContainerManager::ContainerProject"
+      case id
+      when "container_project_timeline"
```

Comment:
@dron sorry i missed last character while copying the url https://github.com/ManageIQ/manageiq/blob/master/spec/helpers/application_helper/toolbar_builder_spec.rb#L1818, look for tests for build_toolbar_disable_button method, there is a test for ems_cluster timeline button maybe you can mimic that one for these new buttons: https://github.com/ManageIQ/manageiq/blob/master/spec/helpers/application_helper/toolbar_builder_spec.rb#L2013